Bottom: b216f34a27ab559e4171e7e53576be3bb2d06284
Top:    ed4180e54ac83285d51d0e0873b34f63ab8c97fa
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 12:41:57 +0000

Add reversed orientation field to face3d structure


---

diff --git a/src/hid/common/face3d.h b/src/hid/common/face3d.h
index 4838c43..95232a6 100644
--- a/src/hid/common/face3d.h
+++ b/src/hid/common/face3d.h
@@ -1,5 +1,6 @@
 typedef struct {
   double nx, ny, nz; /* Face normal?*/
+  bool surface_orientation_reversed;
   GList *contours;
 
   /* For cylindrical surfaces */
